tap_sftp/client.py

```diff
@@ -75,6 +91,10 @@ def __exit__(self):
 
     def close(self):
         if self.__active_connection:
+            # get 'socket' to set the timeout
+            socket = self.sftp.get_channel()
+            # set request timeout to 'None' ie. default value
+            socket.settimeout(None)
```

Comment:
@harshpatel4crest Why do you have to set the timeout manually? Why not using the same request_timeout parameter?

Reply:
Here, during closing the connection, the timeout is set to None.

Reply:
Please use the default value/configured value provided in the tap. Request timeout should never have None value

Reply:
Removed the code as during the connection we are setting the desired timeout value.